spikeinterface/core/segmentutils.py

```diff
@@ -264,5 +267,43 @@ def get_unit_spike_train(self, *args, **kwargs):
 def append_sortings(*args, **kwargs):
     return AppendSegmentSorting(*args, **kwargs)
 
-
 append_sortings.__doc__ == AppendSegmentSorting.__doc__
+
+# Q: should this take as input a concatenated recording or just a list of recordings?
+class SplitSegmentSorting(BaseSorting):
+    """Splits a sorting with a single segment to multiple segments
+    based on the given list of recordings (must be in order)
+
+    Parameters
+    ----------
+    parent_sorting : BaseSorting
+        Sorting with a single segment (e.g. from sorting concatenated recording)
+    recording_list : list
```

Comment:
You can call this `recording_or_recording_list`

Reply:
Done